mopidy/core/tracklist.py

```diff
@@ -209,7 +209,7 @@ def eot_track(self, tl_track):
         :type tl_track: :class:`mopidy.models.TlTrack` or :class:`None`
         :rtype: :class:`mopidy.models.TlTrack` or :class:`None`
         """
-        deprecation.warn("core.tracklist.eot_track", pending=True)
+        deprecation.warn("core.tracklist.eot_track")
```

Comment:
The docstrings should be updated with deprecation information, e.g. between the free form text and the `:param:` lines, add something like:

``` rst
.. deprecated:: 3.0
    Use :meth:`get_eot_tlid` instead.
```